### ISSUE TITLE
Extraneous HTML tags read by screen readers in some areas.

### ISSUE THREAD

**Issue by TheQuinbox** [NationalSecurityAgency/ghidra#6176]
**Describe the bug**
In some areas of Ghidra, the analyzers panel being one of the most noticeable, screen readers will read things like "Less html greater" before the names of some controls.
**To Reproduce**
1. Start NVDA.
2. Open Ghidra.
3. Analyze an executable.
4. Control+tab past the analyzers list (you cannot use standard tab here).
5. Navigate to some of the check boxes, and try to read them with NVDA. 

**Expected behavior**
The checkbox's accessibility info shouldn't have any unescaped HTML in it, like this: check box  checked  \<HTML>Apply any recovered function signature calling convention
I'd like it to read like: check box  checked  Apply any recovered function signature calling convention.

**Environment (please complete the following information):**
 - OS: Windows 10 22H2 (AMD64) build 19045.3803
 - Java Version: 21
 - Ghidra Version: ghidra          10.4-20230928
 - Ghidra Origin: Scoop for Windows.

**Comment by dragonmacher** [comment]
We see the issue and have a fix in progress.
